test/connectors/Acc45ConnectorSpec.scala

```diff
@@ -56,9 +54,10 @@ class Acc45ConnectorSpec extends SpecBase {
 
       "request is not successful due to business error - 'Request could not be processed'" in new Setup {
 
-        when[Future[HttpResponse]](mockHttpClient.POST(any, any, any)(any, any, any, any))
-          .thenReturn(Future.successful(HttpResponse(CREATED, casResponseStr02)))
-
+        when(requestBuilder.withBody(any())(any(), any(), any())).thenReturn(requestBuilder)
+        when(requestBuilder.setHeader(any[(String, String)]())).thenReturn(requestBuilder)
+        when(mockHttpClient.post(any)(any)).thenReturn(requestBuilder)
+        when(requestBuilder.execute(any, any)).thenReturn(Future.successful(HttpResponse(CREATED, casResponseStr02)))
         running(app) {
```

Comment:
I think we should add a new line between the when block and the running(app) just for readability and separation of interests.